aws_lambda/artifacthub-pkg.yml

```diff
@@ -0,0 +1,197 @@
+# Artifact Hub package metadata file
+# https://github.com/artifacthub/hub/blob/master/docs/metadata/artifacthub-pkg.yml
+version: 1.0.0
+name: AWS Lambda Integration
+displayName: AWS Lambda Integration
+createdAt: 2022-05-10T00:00:00Z
+description: Use AWS Lambda with Keptn
+logoURL: https://raw.githubusercontent.com/keptn-contrib/artifacthub/main/aws_lambda/1.0.0/assets/Lambda.svg
+digest: 2022-05-10T00:00:00Z
+license: Apache-2.0
+homeURL: https://keptn.sh/docs/integrations/
+keywords:
+  - keptn
+  - aws
+links:
+  - name: Source
+    url: https://aws.amazon.com/lambda
+recommendations:
+  - url: https://artifacthub.io/packages/helm/keptn/keptn
+annotations:
+  keptn/kind: "webhook"
+  keptn/version: "0.10.0-0.15.0"
+install: |
```

Comment:
I recommend moving away from reusing the text from the README inside the installation section. I don't think it adds any additional value and makes the integration harder to maintain.

Reply:
Fully agree. Please only use `install` section for actual installation instructions (e.g., `helm install ...`  for kubernetes based integrations). For webhooks, it's enough to go through the README.

Reply:
Makes sense. I'm sure we had a conversation in the past about that it was good to paste from readme into install section, but I can't find the discussion. Regardless, less duplication is good in my books. I'll remove the install section from all the integrations I manage (and open PRs) to fix up. So don't merge this until it's all ready.